### PR TITLE
Add header view from https://github.com/rebus007/Header-View

### PR DESCRIPTION
**Pull request by rebus007**
Modified sample adding in the navigation drawer the Header View created by me (https://github.com/rebus007/Header-View)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -12,5 +12,8 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        maven {
+            url 'http://dl.bintray.com/raphaelbussa/maven'
```
Comment (jaredsburrows) [inline build.gradle]: I do not think @chrisbanes will like this. Just upload your project to `jcenter()`.

File: .idea/.name [hunk 2]
```diff
@@ -0,0 +1 @@
+Cheesesquare
```
Comment (jaredsburrows) [inline .idea/.name]: @rebus007 You have a dirty pull request. Do not include user project files.

**Comment by rebus007** [comment]
@jaredsburrows the lib is now in jcenter

**Comment by WebnetMobile** [comment]
What is the point of this PR? This change is irrelevant in scope of cheesesquare project which is about design library mostly. It's not expected to be fully functional app template and your View brings nothing here really.

**Comment by WebnetMobile** [comment]
Also this is bad PR as you pollute the repo with content of your `.idea/` folder -> `+2,312 −35`

**Comment by fernandomantoan** [comment]
@WebnetMobile Agreed.

**Comment by jaredsburrows** [comment]
@WebnetMobile  +1

Also this has conflicts now

**Pull request closed without merging**